AVPlayerExample/ViewController.m

```diff
@@ -195,31 +196,39 @@ - (void)prepareMedia {
     [self startPreview];
 }
 
-- (void)setupAudioSession {
-    // In this example we don't want TwilioVideo to dynamically configure and activate / deactivate the AVAudioSession.
-    // Instead we will setup audio once, and deal with activation and de-activation manually.
-    [[TVIAudioController sharedController] configureAudioSession:TVIAudioOutputVideoChatDefault];
+- (void)startAudioDevice {
+    if (!self.audioDevice) {
+        self.audioDevice = [TVIDefaultAudioDevice audioDeviceWithBlock:^{
+            kDefaultAVAudioSessionConfigurationBlock();
 
-    // This is similar to when CallKit is used, but instead we will activate AVAudioSession ourselves.
-    NSError *error = nil;
-    [[AVAudioSession sharedInstance] setActive:YES error:&error];
-    if (error) {
-        [self logMessage:[NSString stringWithFormat:@"Couldn't activate AVAudioSession. %@", error]];
-    }
+            AVAudioSession *session = [AVAudioSession sharedInstance];
 
-    [[TVIAudioController sharedController] startAudio];
+            NSError *error = nil;
+            if ([session respondsToSelector:@selector(setCategory:mode:options:error:)]) {
+                if (![session setCategory:AVAudioSessionCategoryPlayAndRecord
+                                     mode:AVAudioSessionModeVideoChat
+                                  options:AVAudioSessionCategoryOptionAllowBluetooth | AVAudioSessionCategoryOptionMixWithOthers
+                                    error:&error]) {
+                    NSLog(@"AVAudioSession setCategory:options:mode:error: %@",error);
+                }
+            } else {
+                if (![session setCategory:AVAudioSessionCategoryPlayAndRecord
+                              withOptions:AVAudioSessionCategoryOptionAllowBluetooth
+                                    error:&error]) {
+                    NSLog(@"AVAudioSession setCategory:withOptions %@",error);
+                }
+
+                if (![session setMode:AVAudioSessionModeVideoChat error:&error]) {
+                    NSLog(@"AVAudioSession setMode %@",error);
+                }
+            }
+        }];
+    }
```

Comment:
What is the benefit of providing the custom configuration block since the default is already VideoChat/PlayAndRecord? It seems like it would be sufficient to just use the enabled/disabled feature.